pkgs/development/ruby-modules/gem/default.nix

```diff
@@ -252,6 +252,16 @@ stdenv.mkDerivation ((builtins.removeAttrs attrs ["source"]) // {
     runHook postInstall
   '';
 
+  # For Ruby-generated binstubs, shebang paths are already in Nix store but for
+  # ruby used to build the package. Update them to match the host system. Note
+  # that patchShebangsAuto ignores scripts where shebang line is already in Nix
+  # store.
+  fixupPhase = attrs.fixupPhase or ''
+    runHook preFixup
+    patchShebangs --update --host $out/${ruby.gemPath}/bin
```

Comment:
I think cross bunlder fails here 

```
WARNING:  You build with buildroot.
  Build root: /
  Bin dir: /nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/lib/ruby/gems/3.1.0/bin
  Gem home: /nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/lib/ruby/gems/3.1.0
  Plugins dir: /nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/lib/ruby/gems/3.1.0/plugins
Successfully installed nori-2.7.0
1 gem installed
/nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/lib/ruby/gems/3.1.0 /build
removed 'cache/nori-2.7.0.gem'
removed directory 'cache'
/build
Running phase: fixupPhase
patching script interpreter paths in /nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/lib/ruby/gems/3.1.0/bin
find: '/nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/lib/ruby/gems/3.1.0/bin': No such file or directory
rewriting symlink /nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu/nix-support/gem-meta/spec to be relative to /nix/store/9w2qa550xp0pd002vw4z6hi6cfbyh522-ruby-x86_64-unknown-linux-gnu3.1-nori-2.7.0-x86_64-unknown-linux-gnu
```

Reply:
> bunlder

Uh, I assume *builder* since I can build `pkgsCross.gnu64.bundler`? See also https://github.com/NixOS/nixpkgs/pull/270242#issuecomment-2142433150

I’m not really sure what is the build failure you are referring to. That said, yes, probably a good idea to check whether the directory exists to avoid adding noise to the logs.

Reply:
Done.